src/app/feedback/feedback-widget/feedback-widget.component.ts

```diff
@@ -0,0 +1,74 @@
+import {Component, OnInit, ElementRef} from '@angular/core';
+import {AngularFireModule, AuthProviders, AuthMethods, AngularFire} from 'angularfire2';
+import {PresentationComponent} from "../../presentation/presentation/presentation.component";
+@Component({
+  selector: 'app-feedback-widget',
+  templateUrl: './feedback-widget.component.html',
+  styleUrls: ['./feedback-widget.component.css']
+})
+export class FeedbackWidgetComponent implements OnInit {
+  open = false;
+  email: string = '';
```

Comment:
You can rely on implicit types if you are initializing these strings to empty string.
Why are you initializing them to empty string?